packages/documentation/src/components/dashboard/DashboardDataFetch.jsx

```diff
@@ -142,22 +149,28 @@ export async function deploysFetch(
 
     // If commit wasn't deployed by a full build, check for single-app build
     let hasSuccessfulSingleAppBuild = false;
+    let isContinuousDeploymentEnabled = false;
     if (isVetsWebsite && (!isOnDev || !isOnStaging || !isOnProd)) {
       const buildArtifactUrl = `${repo.buildArtifacts}/${sha}.txt`;
       const response = await fetch(buildArtifactUrl);
       if (response.ok) {
         const fileText = await response.text();
-        const matchText = fileText.match(/IS_SINGLE_APP_BUILD=(\w+)/)[1];
-        const isSingleAppBuild = matchText === 'true';
+        const isSingleAppBuild =
+          getValueFromFile(fileText, 'IS_SINGLE_APP_BUILD') === 'true';
         const workflowSucceeded = workflowConclusions[sha] === 'success';
         hasSuccessfulSingleAppBuild = isSingleAppBuild && workflowSucceeded;
+        isContinuousDeploymentEnabled =
+          getValueFromFile(fileText, 'IS_CONTINUOUS_DEPLOYMENT_ENABLED') ===
+          'true';
       }
     }
 
     results[sha] = {
       dev: isOnDev || hasSuccessfulSingleAppBuild,
       staging: isOnStaging || hasSuccessfulSingleAppBuild,
-      prod: isOnProd || hasSuccessfulSingleAppBuild,
+      prod:
+        isOnProd ||
+        (isContinuousDeploymentEnabled && hasSuccessfulSingleAppBuild),
```

Comment:
I reflected on it a bit, and I agree with having two separate values for these. It will help for debugging, and also if we want to display the individual values on the dashboard at some point

Reply:
It's ironic you brought that up because I almost combined them into one value haha. I agree, having them as separate values gives us more flexibility